todo_app/app.py

```diff
@@ -1,14 +1,26 @@
-from flask import Flask
+from flask import Flask, render_template, request
 
 from todo_app.flask_config import Config
 
+from todo_app.data import session_items
+
+
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    itemList = session_items.get_items()
```

Comment:
Python naming convention for variable names is snake_case. So: `item_list`

Reply:
done